tests/storage/storage.go

```diff
@@ -1460,8 +1480,42 @@ var _ = SIGDescribe("Storage", func() {
 				err = virtClient.VirtualMachineInstance(testsuite.GetTestNamespace(vmi)).Delete(context.Background(), vmi.ObjectMeta.Name, &metav1.DeleteOptions{})
 				Expect(err).ToNot(HaveOccurred(), failedDeleteVMI)
 				libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 180)
-			})
+			},
+				Entry("PVC source", addPVCLunDisk),
+				Entry("DataVolume source", addDataVolumeLunDisk),
+			)
+
+			It("should run the VMI created with a DataVolume source", func() {
+				sc, foundSC := libstorage.GetBlockStorageClass(k8sv1.ReadWriteOnce)
+				if !foundSC {
+					Skip("Unable to find valid storage class")
+				}
+				pv, err = tests.CreatePVwithSCSIDisk(sc, "scsipv", nodeName, device)
+				Expect(err).ToNot(HaveOccurred())
+				dv := libdv.NewDataVolume(
+					libdv.WithBlankImageSource(),
+					libdv.WithPVC(libdv.PVCWithStorageClass(sc),
+						libdv.PVCWithBlockVolumeMode(),
+						libdv.PVCWithAccessMode(k8sv1.ReadWriteOnce),
+						libdv.PVCWithVolumeSize("8Mi"),
+					),
+				)
+				dv, err = virtClient.CdiClient().CdiV1beta1().DataVolumes(testsuite.GetTestNamespace(nil)).Create(context.Background(), dv, metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred())
 
+				By("Creating VMI with LUN disk")
+				vmi := libvmi.NewAlpine()
+				addDataVolumeLunDisk(vmi, "lun0", dv.Name)
```

Comment:
I think it would be good to show that lun disk in use somehow.  Either as the boot disk or login and create a filesystem on the blank disk and use it